apps/financial-aid/backend/src/environments/environment.ts

```diff
@@ -40,7 +41,8 @@ const devConfig = {
   files: {
     cloudFrontPublicKeyId: process.env.CLOUDFRONT_PUBLIC_KEY_ID ?? '',
     cloudFrontPrivateKey: process.env.CLOUDFRONT_PRIVATE_KEY ?? '',
-    fileBaseUrl: 'https://fjarhagsadstod.dev.sveitarfelog.net/files',
+    fileBaseUrlOsk: 'https://fjarhagsadstod.dev.sveitarfelog.net/files',
+    fileBaseUrlVeita: 'https://fjarhagsadstod.dev.sveitarfelog.net/files',
```

Comment:
Should this be the same links? Then why have them twice? 👯‍♀️

Reply:
This a change that was here when I took over the PR, but great question, will change, this doesn't make sense